.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+
+## Code Review Checklist
+ * Does the PR describe what changes are being made?
```

Comment:
I think this should say: "Does the PR describe what changes are being made and why?".  I think its really important we all understand **why** we are doing what we do.

Reply:
good call